lib/Less/Functions.php

```diff
@@ -813,11 +813,11 @@ private function _isa($n, $type) {
 		return is_a($n, $type) ? new Less_Tree_Keyword('true') : new Less_Tree_Keyword('false');
 	}
 
-	public function tint($color, $amount) {
+	public function tint($color, $amount = null) {
```

Comment:
Actually the assumption was fine. In fact that "50%" in `less.js` implementation is not something really intended in `tint`/`shade` (in fact neither default value makes any fair sense there) but a direct consequence of the author of those functions just did not specify the argument (probably for the sake of less verbosity) and therefore [`undefined` amount](https://github.com/less/less.js/blob/master/lib/less/functions/color.js#L312) parameter in `tint`/`shade` explicitly maps to `50%` amount in `mix` (where it makes perfect sense).

In other words, the documentation only reflects simplified/weird implementation rather than really something reasonable. (So `$amount = null` would reflect exactly what they do in `less,js`).